job_templates/sag_nemo/info.conf

```diff
@@ -0,0 +1,5 @@
+{
+    description = "scatter & gather workflow using pytorch + ddp"
```

Comment:
"Scatter and Gather Workflow for NeMo"

Reply:
fixed

Reply:
This "sag_nemo" has not been updated, the other yes.

Reply:
need to update "job_templates/sag_nemo/info.conf" as well

Reply:
ok. I missed that.